playground/TestShop/BasketService/Repositories/RedisBasketRepository.cs

```diff
@@ -36,7 +36,7 @@ public IEnumerable<string> GetUsers()
             return null;
         }
 
-        return JsonSerializer.Deserialize<CustomerBasket>(data!, s_jsonSerializerOptions);
+        return JsonSerializer.Deserialize<CustomerBasket>(data.ToString(), s_jsonSerializerOptions);
```

Comment:
I updated to new VS and got a warning about ambigious overload here between span and string. I believe this is caused by work in the compiler to make span more first class.

Fixed by explicitly calling ToString instead of previously using implicitly string conversion.